spec/features/user_visits_trips_index_sees_first_30_trips_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe 'User visits trip index' do
+  it 'should display first 30 trips and link to see more' do
+    trips = create_list(:trip, 35)
+
+    visit trips_path
+
+    expect(page).to have_content(trips.first.duration)
+    expect(page).to have_content(trips.first.start_date)
+    expect(page).to have_content(trips.first.start_station)
+    expect(page).to have_content(trips.first.end_date)
+    expect(page).to have_content(trips.first.end_station)
+    expect(page).to have_content(trips.first.bike_id)
+    expect(page).to have_content(trips.first.subscription_type)
+    expect(page).to have_content(trips.first.zip_code)
+
+    expect(page).to have_content(trips[29].bike_id)
+    expect(page).to_not have_content(trips[30].bike_id)
```

Comment:
I really like this test structure. Test that one has all the attributes, then test that a couple random bikes are also there.